deployments/image/configmap/Dockerfile

```diff
@@ -46,8 +46,8 @@ ENV DISPATCHER_TIMEOUT "300s"
 ENV KUBE_CLIENT_QPS "1000"
 ENV KUBE_CLIENT_BURST "1000"
 ENV PREDICATES ""
-ENV OPERATOR_PLUGINS "general"
 ENV ENABLE_CONFIG_HOT_REFRESH "true"
+ENV OPERATOR_PLUGINS "general,spark-k8s-operator"
```

Comment:
this change will build the docker image by default with spark-k8s-operator plugin enabled. 
We might want to provide an option instead of doing this by default. I think we can add the env var  `OPERATOR_PLUGINS` in the helm chart template: https://github.com/apache/incubator-yunikorn-release/blob/2ade62ac43f9f62f804739f418fcdcb256b9703b/helm-charts/yunikorn/templates/deployment.yaml#L55.  And expose this in https://github.com/apache/incubator-yunikorn-release/blob/master/helm-charts/yunikorn/values.yaml.

Reply:
Sure, I'll roll back this line to include only the general plugin by default. I'll create another JIRA to track the work on exposing it in the Helm chart

Reply:
Created https://issues.apache.org/jira/browse/YUNIKORN-642 to track the follow-up work